src/EnergyPlus/WeatherManager.cc

```diff
@@ -281,6 +282,11 @@ namespace WeatherManager {
 	Array1D< SpecialDayData > SpecialDays;
 	Array1D< DataPeriodData > DataPeriods;
 
+	std::shared_ptr< BaseGroundTempsModel > siteShallowGroundTempsPtr;
```

Comment:
@mmatts Are these used anywhere outside of this file? Do they need to be shared_ptrs? It doesn't seem like they need to be.